### PR TITLE
Admin bar improvements

### LINKED ISSUE #428: Adminbar followup
- [x] Add CHANGELOG entry
- [x] Document constant in new README
- [x] Add a `title` to the metrics node that includes labels, like `Hit ratio: x, Hits: x, Misses: x, etc.`
- [x] When the plugin is "not connected" (error), let's make the node red (QM does this) and show the status where the metrics are when it's connected.
- [x] When the object cache is disabled, let's just show the status where the metrics would be.

### PR DESCRIPTION
**Pull request by A5hleyRich**
Resolves #428 

https://user-images.githubusercontent.com/1422996/223417430-b9b176b3-e81b-4329-bb8d-c59b0f423674.mp4

### CODE REVIEW

File: includes/class-plugin.php [hunk 1]
```diff
@@ -665,15 +665,19 @@ public function render_admin_bar( $wp_admin_bar ) {
...
         }
 
+        $redis_status = $this->get_redis_status();
+
         $wp_admin_bar->add_node(
             [
                 'id' => 'redis-cache',
                 'title' => __( 'Object Cache', 'redis-cache' ),
+                'meta' => [
+                    'html' => '<style>#wpadminbar ul li.redis-cache-error { background: #c00; } #wpadminbar:not(.mobile) .ab-top-menu>li.redis-cache-error:hover>.ab-item { background: #b30000; color: #fff; }</style>',
```
Comment (A5hleyRich) [inline includes/class-plugin.php:675]: @tillkruss this isn't elegant, but it's the only way we can color the node without loading an external CSS file. We could inject the style into the head, but there doesn't seem any benefit.

Comment (tillkruss) [reply includes/class-plugin.php:675]: It seems efficient.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -47,18 +47,19 @@ The plugin comes with vast set of configuration options.
...
+| `WP_REDIS_GLOBAL_GROUPS`             | `[]`    | Additional groups that are considered global on multisite networks       |
+| `WP_REDIS_METRICS_MAX_TIME`          | `3600`  | The maximum number of seconds metrics should be stored                   |
+| `WP_REDIS_IGBINARY`                  | `false` | Whether to use the igbinary PHP extension for serialization              |
+| `WP_REDIS_SERIALIZER`                |         | The PhpRedis/Relay serializer used for data serialization                |
+| `WP_REDIS_DISABLED`                  | `false` | Emergency switch to bypass the object cache without deleting the drop-in |
+| `WP_REDIS_DISABLE_ADMINBAR`          | `false` | Disables admin bar display                                               |
+| `WP_REDIS_DISABLE_METRICS`           | `false` | Disables metrics collection and display                                  |
+| `WP_REDIS_DISABLE_BANNERS`           | `false` | Disables promotional banners                                             |
+| `WP_REDIS_DISABLE_DROPIN_AUTOUPDATE` | `false` | Disables the drop-in auto-update                                         |
```
[the remark this answers is not in the document]
Comment (A5hleyRich) [reply README.md]: Sorry about that. PhpStorm likes to auto-format everything.

File: includes/class-plugin.php (hunk 1 above)
Comment (tillkruss) [reply includes/class-plugin.php:675]: It does 🫠

**Review by tillkruss — APPROVED** [review]
Beauty!

**Pull request merged**